test/test_nn.py

```diff
@@ -3728,18 +3728,28 @@ def forward(self, *args):
         module_name='PoissonNLLLoss',
         input_size=(2, 3, 4, 5),
         target_fn=lambda: torch.randn(2, 3, 4, 5).floor_().abs_(),
-        desc='reduced_loss',
+        desc='no_full_loss', # without sterling approx
     ),
     dict(
         module_name='PoissonNLLLoss',
-        constructor_args=(False, True, True),
+        constructor_args=(False, True, True, 1e-8, True),
```

Comment:
Nit: This didn't need to be changed (since both 1e-8 and reduce=True are default arguments)